OCP-Infrastructure/kafka/install_kafka.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-oc apply -f .
\ No newline at end of file
+oc apply -f . -n hogarama-kafka
```

Comment:
hogarama, oder? ist ja nicht in eigenem namespace

Reply:
Das benötige ich eigentlich nicht. Habe überlegt ob es für den Play-Cluster noch so verwendet werden könnte, oder gleich löschen. Ich tendiere eher zum Löschen und stattdessen verwenden von `install.sh`

Reply:
dann würde ich auch löschen sagen. play-cluster wird demnächst ja neu aufgesetzt und da würde ich, wenn hoarama darauf installiert werden soll, auch auf den einfacheren prozess als auf den alten setzen